src/style.css

```diff
@@ -0,0 +1,117 @@
+:root {
+  --cardMainColor: #060b35;
+  --cardSecondarycolor: #616070;
+  --buttonColor: #00acdc;
+}
+
+.page {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 166px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  font-family: "Roboto", helvetica, sans-serif;
+  border: 1px solid #f3f3f3 ;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  object-position: center;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--cardMainColor);
+  padding: 0 0 4px;
```

Comment:
you can reset padding for tag and just set padding for one side

Reply:
don't forget about css properties order
http://codeguide.academy/html-css.html#css-order